integration/setup/tenant_cluster.go

```diff
@@ -71,6 +71,11 @@ func EnsureTenantClusterDeleted(ctx context.Context, id string, config Config, w
 
 	config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("deleting tenant cluster %#q", id))
 
+	err = config.Guest.Initialize()
```

Comment:
Is this something new?

Reply:
Without this the guest framework clients don't work. It needs to read the secret generated by cert-operator in the minikube cluster.


https://github.com/giantswarm/e2e-harness/blob/b2048d8645c15ac583a270c57b95855426e48958/pkg/framework/guest.go#L105